### PR TITLE
Shrink CHANGELOG

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,58 +4,62 @@
...
 - Bug #178: Exclude parent role from `Manager::getAllChildRoles()` (@arogachev)
 - Enh #134: Improve handling and control of `Assignment::$createdAt` (@arogachev)
 - Chg #134: Add `$createdAt` parameter to `ManagerInterface::assign()` (@arogachev)
-- Chg #134: Replace parameters with `$assignment` parameter in `AssignmentsStorageInterface::add()` (@arogachev)
-- Enh #165: Improve performance (@arogachev)
+- Chg #134: Replace all parameters with single `$assignment` parameter in `AssignmentsStorageInterface::add()` 
+  (@arogachev)
+- Enh #165, #206: Improve performance:
+  - Optimize calls for getting child items within the loops.
+  (@arogachev)
```
Comment (vjik) [inline CHANGELOG.md:15]:
```suggestion
- Enh #165, #206: Improve performance: optimize calls for getting child items within the loops (@arogachev)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,58 +4,62 @@
...
+- Enh #165, #203, #206, #208, #237: Add methods to `ItemsStorageInterface`:
     - `roleExists()`;
     - `getRolesByNames()`;
     - `getPermissionsByNames()`;
     - `getAllChildren()`;
     - `getAllChildRoles()`;
     - `getAllChildPermissions()`;
     - `hasChild()`;
-    - `hasDirectChild()`.
+    - `hasDirectChild()`;
```
Comment (vjik) [inline CHANGELOG.md:25]: Need check correctly work of Yii Dev Tool with lists in changelog

Comment (arogachev) [reply CHANGELOG.md:25]: https://github.com/yiisoft/yii-dev-tool/issues/290

Comment (arogachev) [reply CHANGELOG.md:25]: For now, I decided to not use lists at all.

File: CHANGELOG.md (hunk 1 above)
Comment (arogachev) [reply CHANGELOG.md]: I removed the list, but rewrote it a bit, because it's not the only one change.

**Pull request merged**